test/test_unit.jl

```diff
@@ -238,8 +238,8 @@ Cassette.@context Ctx
       @test isapprox(Trixi.multiply_dimensionwise(matrix, data_in), [3.0 4.0 5.0 6.0])
 
       n_vars   = 3
-      size_in  = 4
-      size_out = 5
+      size_in  = 2
+      size_out = 3
```

Comment:
out of curiosity: why change?

Reply:
Same reason as before: Reduce the size to speed-up CI time. These tests using LoopVectorization with coverage reports are quite expensive in CI...